templates/common/infra/terraform/core/database/postgresql/postgresql_output.tf

```diff
@@ -0,0 +1,16 @@
+output "AZURE_POSTGRESQL_DATABASE_NAME" {
+  value     = azurerm_postgresql_flexible_server_database.database.name
+  sensitive = true
+}
+
+output "AZURE_POSTGRESQL_FQDN" {
+  value = azurerm_postgresql_flexible_server.psqlServer.fqdn
+}
+
+output "AZURE_POSTGRESQL_SPRING_DATASOURCE_URL" {
+  value = "jdbc:postgresql://${azurerm_postgresql_flexible_server.psqlServer.fqdn}:5432/${azurerm_postgresql_flexible_server_database.database.name}?sslmode=require"
```

Comment:
This seems fragile to me.  Is that port guaranteed to be the same?

Reply:
> This seems fragile to me. Is that port guaranteed to be the same?

Yep, the port of Azure Database for PostgreSQL is 5432. We can't modify it.